### PR TITLE
session2\chapter2lightning-in-action

### PR DESCRIPTION
**Pull request by cherishlihong**

### CODE REVIEW

File: session2/chapter2/lightning-in-action.md [hunk 1]
```diff
@@ -0,0 +1,1209 @@
+# 快速开始
+## **警告**
+* TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+*  若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致延迟。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
* 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致延迟。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
```

File: session2/chapter2/lightning-in-action.md [hunk 2]
```diff
@@ -0,0 +1,1209 @@
...
+```
+.../tidb-ansible/resouce/bin/tidb-lightning-ctl -switch-mode=normal
+```
+## **数据库权限要求**
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md:10]: Prefer to add a blank line before L8.

File: session2/chapter2/lightning-in-action.md [hunk 3]
```diff
@@ -0,0 +1,1209 @@
...
+TiDB Lightning 需要下游 TiDB 具有如下权限：
+
+| 权限 | SELECT | INSERT | UPDATE | DELETE | CREATE | DROP | ALTER | 
+|:----:|:----:|:----:|:----:|:----:|:----:|:----:|:----:|
+| 作用域 | Tables | Tables | Tables | Tables | Databases,   tables | Databases,   tables | Tables |
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
| 作用域 | Tables | Tables | Tables | Tables | Databases, tables | Databases, tables | Tables | 
```

File: session2/chapter2/lightning-in-action.md [hunk 4]
```diff
@@ -0,0 +1,1209 @@
...
+|:----:|:----:|:----:|:----:|:----:|:----:|:----:|:----:|
+| 作用域 | Tables | Tables | Tables | Tables | Databases,   tables | Databases,   tables | Tables | 
+
+  如果配置项 `checksum = true`，则 TiDB Lightning 需要有下游 TiDB admin 用户权限。
+
+##  
+## 硬件需求
+tidb-lightning` 和 `tikv-importer` 这两个组件皆为资源密集程序，建议各自单独部署。
+
+为了优化效能，建议硬件配置如下：
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
为了保证导入性能，建议最低硬件配置如下：
```

File: session2/chapter2/lightning-in-action.md [hunk 5]
```diff
@@ -0,0 +1,1209 @@
...
+
+`tikv-importer`
+
+  * 32+ 逻辑核 CPU
+  * 40 GB+ 内存
+  * 1 TB+ SSD 硬盘，IOPS 越高越好（要求 ≥8000）
+
+        * 硬盘必须大于最大的 N 个表的大小总和，其中 N = max(index-concurrency, table-concurrency)。
+
+  *  使用万兆网卡，带宽需 300 MB/s 以上
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
  * 使用万兆网卡，带宽需 300 MB/s 以上
```

File: session2/chapter2/lightning-in-action.md [hunk 6]
```diff
@@ -0,0 +1,1209 @@
...
+$ ansible-playbook deploy.yml -l IS1，LS
+或者
+$ ansible-playbook deploy.yml --tags=lightning
+```
+**Step5：** 启动 Importer 以及 Lightning
+
+注意，必须先启动 Importer ，再启动 Lightning ，顺序不能换。
+
+1. 登录到部署 Importer 以及 Lightning 的服务器
+1. 进入部署目录
```
Comment (lonng) [inline session2/chapter2/lightning-in-action.md]: Duplicate number

File: session2/chapter2/lightning-in-action.md [hunk 7]
```diff
@@ -0,0 +1,1209 @@
+# 快速开始
+## **警告**
+* TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+*  若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致延迟。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
+```
```
Comment (GregoryIan) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
  ```
```

File: session2/chapter2/lightning-in-action.md [hunk 8]
```diff
@@ -0,0 +1,1210 @@
+# 快速开始
+## **警告**
+* TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+* 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致延迟。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
+
+.../tidb-ansible/resouce/bin/tidb-lightning-ctl -switch-mode=normal
```
Comment (GregoryIan) [inline session2/chapter2/lightning-in-action.md]:
```suggestion
  ```
  .../tidb-ansible/resouce/bin/tidb-lightning-ctl -switch-mode=normal
  ```
```

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**